extensions/jaxb/deployment/src/main/java/io/quarkus/jaxb/deployment/JaxbProcessor.java

```diff
@@ -169,6 +177,18 @@ void processAnnotationsAndIndexFiles(
                     proxyDefinitions.produce(new NativeImageProxyDefinitionBuildItem(className, Locatable.class.getName()));
                     addReflectiveClass(reflectiveClass, true, false, className);
                 });
+        produceProxyIfExist(proxyDefinitions, "com.sun.xml.bind.marshaller.CharacterEscapeHandler");
+        produceProxyIfExist(proxyDefinitions, "com.sun.xml.internal.bind.marshaller.CharacterEscapeHandler");
+        produceProxyIfExist(proxyDefinitions, "org.glassfish.jaxb.core.marshaller.CharacterEscapeHandler");
+        produceProxyIfExist(proxyDefinitions, "com.sun.xml.txw2.output.CharacterEscapeHandler");
+        produceProxyIfExist(proxyDefinitions, "org.glassfish.jaxb.characterEscapeHandler");
+        produceProxyIfExist(proxyDefinitions, "org.glassfish.jaxb.marshaller.CharacterEscapeHandler");
+
+        proxyDefinitions.produce(new NativeImageProxyDefinitionBuildItem("com.sun.xml.txw2.TypedXmlWriter"));
+        Set<String> proxiesCreated = new HashSet<>();
+        DotName typedXmlWriterDN = DotName.createSimple("com.sun.xml.txw2.TypedXmlWriter");
+        // getAllKnownDirectImplementors skip interface, so use own recursion
+        produceRecursiveProxies(index, typedXmlWriterDN, proxyDefinitions, proxiesCreated);
```

Comment:
Not sure I understand what you're trying to do here. Could you explain?

Reply:
Cxf use jaxb xmlSchemaGeneratorwhich can only works with proxy...

Reply:
and all inherited class of TypedXmlWriter use proxy. I just avoid to maintain a big list of class. But If you prefer, I have the list of class.